src/main.cpp

```diff
@@ -35,6 +35,9 @@ int WINAPI modengine_entrypoint(void)
         DebugBreak();
     }
 
+    auto logger = logging::setup_logger("", TRUE);
```

Comment:
This should be moved back until after settings are loaded (or alternatively it can use a well defined path for log files like LocalAppData and infer the `debug` option from the `MODENGINE_DEBUG_GAME` var being present).